manual/manual.tex

```diff
@@ -307,6 +314,25 @@ \section{Runtime Configuration\label{runtime-config}}
 yield point, instead of sampling the time. This has a higher runtime
 overhead than \texttt{velocity} and we do not recommend it.
 
+\subsection{Stack behavior \texttt{(.stackSize and .restoreFrames)} }
+Stopify supports heap bounded stacks for programming languages that want to
+support deeply recursive code and not inherit the stack size limit imposed
+by JavaScript VMs. Stopify accomplishes by restoring only
+a suffix of a captured stack and saving the rest of it for the next restore
+event. Once a program finishes running, Stopify checks to see if
+there is a prefix of the stack that was previously saved and restores such a
+prefix.
+\\ \\
+\texttt{.stackSize}: Controls the maximum number of stack frames that a stopified
+program can consume before it must suspend. A constant value implies support for heap
```

Comment:
"...before it must ~~suspend~~ reify the current stack on the heap and continue with a fresh one." or something like that.